pyproject.toml

```diff
@@ -56,6 +56,8 @@ onnx = "1.16.1"
 onnxscript = "^0.1.0.dev20240806"
 onnxruntime = "^1.19.0"
 pandas = "^2.2.3"
+jupyter = "^1.1.1"
```

Comment:
Please remove the jupyter dependency as it is optional and leads to some dependency overhead that is not required for all users :)